main.go

```diff
@@ -4,58 +4,55 @@ import (
 	log "github.com/sirupsen/logrus"
 
 	"github.com/Lubwama-Emmanuel/Interfaces/app"
-	filesystem "github.com/Lubwama-Emmanuel/Interfaces/storage/file_system"
+	"github.com/Lubwama-Emmanuel/Interfaces/storage/mongodb"
 )
 
 func main() {
 	// storage := memory.NewMemoryStorage()
-	storage := filesystem.NewFileSytemDatabase("data.json")
+	// storage := filesystem.NewFileSytemDatabase("data.json")
+	storage := mongodb.NewMongoDB("mongodb://localhost:27017")
```

Comment:
Now that we are moving to dependencies like DBs that often need complex configs, it is time to introduce [viper](https://github.com/spf13/viper) for configuration management